addons/ofxAssimpModelLoader/scripts/formulas/assimp.sh

```diff
@@ -320,6 +320,9 @@ function copy() {
 	else
 		cp -Rv lib/libassimp.a $1/lib/$TYPE/assimp.a
 	fi
+
+    # Copy License File
+    cp -v LICENSE $1/LICENSE
```

Comment:
There's no need to name the file when copying it here (and in the other formulas). Just do:

    cp -v LICENSE $1/